assets/src/domain/shared/entities/tickets/constants.ts

```diff
@@ -4,11 +4,11 @@ export const MODEL_NAMES = {
 };
 
 export const TICKET_STATUS_ID = {
+	ARCHIVED: 'TKA',
```

Comment:
we can use the term `TRASHED` here, I just don't want to change the status code, sorry for not being clear about that.
It will be a little weird having `TRASHED === TKA` but I would still prefer to keep it like that